modules/cudf/src/column.ts

```diff
@@ -700,6 +701,98 @@ export interface Column<T extends DataType = any> {
    */
   isNotNaN(memoryResource?: MemoryResource): Column<Bool8>;
 
+  /**
+   * Creates a column of `BOOL8` elements indicating strings in which all characters are valid for
+   * conversion to floats.
+   *
+   * @param memoryResource The optional MemoryResource used to allocate the result Column's device
+   *   memory.
+   * @returns A non-nullable column of `BOOL8` elements with `true` representing convertible
+   *   values
+   */
+  isFloat(memoryResource?: MemoryResource): Column<Bool8>;
```

Comment:
String naming should be here too, since we can't call this on other dtypes right?

```suggestion
  stringIsFloat(memoryResource?: MemoryResource): Column<Bool8>;
```